xla/service/gpu/gpu_executable.cc

```diff
@@ -343,6 +365,23 @@ absl::Status ExecuteThunks(const std::string& module_name,
     command_buffer_trace_stream = borrowed_command_buffer_trace_stream->get();
   }
 
+  // Borrow stream for additional compute streams
+  absl::flat_hash_map<ExecutionStreamId, se::Stream*> additional_compute_streams;
+  if (additional_compute_stream_ids.size() > 0) {
+    int num_streams = additional_compute_stream_ids.size();
+    absl::StatusOr<std::vector<StreamPool::Ptr>> additional_streams =
```

Comment:
TF_ASSIGN_OR_RETURN, because if you can't borrow streams that are requested there is no point to continue as thunks will return error anyway